modules/rhcos-enabling-multipath.adoc

```diff
@@ -1,119 +1,50 @@
-kind: MachineConfig
-metadata:
-  labels:
-    machineconfiguration.openshift.io/role: "worker"
-  name: 99-worker-kargs-mpath
-spec:
-  kernelArguments:
-    - rd.multipath=default
-    - root=/dev/disk/by-label/dm-mpath-root
-----
-
-. Create the new machine config by using either the master or worker YAML file you previously created:
+. To enable multipath and start the `multipathd` daemon, run the following command:
 +
 [source,terminal]
 ----
-$ oc create -f ./99-master-kargs-mpath.yaml
+$ mpathconf --enable && systemctl start multipathd.service
 ----
+** Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
 
-. Check the machine configs to see that the new one was added:
-+
-[source,terminal]
-----
-$ oc get MachineConfig
-----
+. Edit the `/etc/multipath.conf` file to target the multipath device directly for installation, either on the kernel command line by using `coreos.inst.install_dev` or interactively by invoking the `coreos-installer` program.
 +
-.Example output
-[source,terminal]
-----
-NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-00-worker                                          52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-master-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-container-runtime                        52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-01-worker-kubelet                                  52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-kargs-mpath                              52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             105s
-99-master-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-master-ssh                                                                                 3.2.0             40m
-99-worker-generated-registries                     52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-99-worker-ssh                                                                                 3.2.0             40m
-rendered-master-23e785de7587df95a4b517e0647e5ab7   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-rendered-worker-5d596d9293ca3ea80c896a1191735bb1   52dd3ba6a9a527fc3ab42afac8d12b693534c8c9   3.2.0             33m
-----
+You can configure multipath so that the names of the multipath devices are of the form `/dev/mapper/mpath` `_n_`. Without this setting, the names of the multipath devices are aliased to a World Wide Name (WWN) symlink, such as `/dev/disk/by-label`.
```

Comment:
Unfortunately, we have no easy way right now for customers to provide custom multipath configs to the installed node on first boot. So they have to stay with the default configs emitted by `mpathconf --enable`.

If there's a single multipath device, then it *should* reliably be `/dev/mapper/mpatha`. If there are multiple multipath devices, or if they want to be more explicit, then using the `/dev/disk/by-id/wwn-<WWN>` symlinks would be best.

Reply:
Forgive my ignorance, please, as I try to understand the steps that the user would take here.

Can you provide an example output that the user would see after running step 1 (`$ mpathconf --enable && systemctl start multipathd.service`)? I'm thinking that could help with step two, where I've changed it to reflect your wording by saying, "Use the default configs emitted by `mpathconf --enable`".

Reply:
Thanks, @jlebon, and pardon my ignorance in this process, please. 😁 

I've made another round of updates to hopefully give clearer directions on what users need to do to append kargs using `coreos-installer install`. So now in Step 2, I've broken it out for instructions to do either: A) A single mpath device, or B) Multiple mpath devices. I also left a note in there that they can use `coreos.inst.install_dev`, though I'm not certain how. If you have a suggestion for what step(s) the user should take for this, let me know and I'll include it. Thanks!